### PR TITLE
Add and update geocoding client text

### PR DESCRIPTION
**Pull request by steveoh**
fill in the gaps

this adds the ugrc api client with downloads
moves the download links from the introduction blog post to the download page
and updates some of the index page items

### CODE REVIEW

File: data/address/index.html [hunk 1]
```diff
@@ -31,19 +31,25 @@
...
     <td>UGRC</td>
-    <td>Custom geocoding http web api</td>
+    <td>The foundational service for all Utah geocoding and query access to the SGID.</td>
   </tr>
   <tr>
-    <td><a href="{% link data/address-geocoders-locators/index.html %}#GeocodingToolbox">Utah Geocoding Toolbox</a></td>
+    <td><a href="{% link data/address-geocoders-locators/index.html %}#OfficialClient">Official UGRC API Client for Geocoding</a></td>
+    <td>Cross-Platform Desktop Geocoding Application</td>
+    <td>UGRC</td>
+    <td>The recommended client and easiest application to use for geocoding Utah addresses.</td>
```
Comment (gregbunce) [inline data/address/index.html:43]: easiest?  I suppose.  Just thinking if you're going to geocode an address in Pro, this wouldn't be the easiest.  but, i know what you're saying. it's the most direct and simple way to geocode a table of addresses.

File: data/address-geocoders-locators/index.html [hunk 2]
```diff
@@ -52,89 +59,32 @@
...
-      <a href="https://developer.mapserv.utah.gov/StartupGuide"
-        >read the getting started guide</a
-      >. There are also examples of geocoding in
-      <a href="https://github.com/agrc/GeocodingSample"
-        >multiple programming languages</a
-      >.
+      <img src="{% link images/api-client/web_api_logo.png %}" width="92" class="inline-text-left" alt="UGRC API logo" loading="lazy"/>The UGRC API Client is a cross-platform desktop geocoding tool designed to carefully and successfully guide you through your geocoding tasks. This app is our best and most user friendly geocoding client. The app will geocode <code>*.csv</code> files. It will require no license or subscription to use. It will automatically download updates as soon as they are available to stay current and add new features.
```
Comment (gregbunce) [inline data/address-geocoders-locators/index.html:69]: i wonder if we should omit the word "best".  maybe we say it's the best for geocoding a table of addresses.  i agree with that.

**Review by gregbunce — DISMISSED** [review]
i made a few minor suggestions/comments, however, i approve this pr regardless of whether you incorporate them or not.

this is a great addition to the locators page and the address index page.  it's nice to see them all in once swoop.  i like it!

File: data/address/index.html (hunk 1 above)
Comment (steveoh) [reply data/address/index.html:43]: You think installing pro, downloading a toolbox, adding the toolbox, and using it to geocode is simpler?

Comment (jacobdadams) [reply data/address/index.html]: TBH, I agree with Steve on this one. I've fully transitioned to using the API client for geocoding. It just feels slicker/smoother/easier. 

I suppose the one thing you lose is the guided workflow in Pro for reviewing matches and fixing unmatched points, but that's easy enough.

File: _posts/2021-11-29-introducing-the-official-ugrc-api-client.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ The official [UGRC API client](https://github.com/agrc/api-client) is now availa
 ## A few highlights we are proud of
 
 - No license is needed to use the client! It is download and go.
-- The client is cross-platform. It works on [MacOS](https://github.com/agrc/api-client/releases/download/v1.6.0/UGRC.API.Client-1.6.0-x64.dmg) and [Windows](https://github.com/agrc/api-client/releases/download/v1.6.0/ugrc-api-client-1.6.0-win32-setup.exe).
+- The client is cross-platform. It works on [MacOS]({% link data/address-geocoders-locators/index.html %}#OfficialClient) and [Windows]({% link data/address-geocoders-locators/index.html %}#OfficialClient).
```
Comment (jacobdadams) [inline _posts/2021-11-29-introducing-the-official-ugrc-api-client.md:30]: We've still got the direct links in the paragraph above this (I can't figure out how to do a suggestion there to fix them).

**Review by stdavis — APPROVED** [review]

File: data/address/index.html (hunk 1 above)
Comment (jacobdadams) [reply data/address/index.html]: mm, I haven't tried using masquerade for geocoding yet.

File: data/address-geocoders-locators/index.html (hunk 2 above)
Comment (jacobdadams) [inline data/address-geocoders-locators/index.html]:
```suggestion
      <img src="{% link images/api-client/web_api_logo.png %}" width="92" class="inline-text-left" alt="UGRC API logo" loading="lazy"/>The UGRC API Client is a cross-platform, stand-alone desktop geocoding tool designed to guide you step-by-step through your geocoding tasks. This app is our best and most user friendly geocoding client for geocoding <code>*.csv</code> files. It doesn't require a license or subscription to use and automatically downloads updates as soon as they are available to stay current and add new features.
```
Just a sentence flow suggestion, take it or leave it.

File: data/address-geocoders-locators/index.html [hunk 4]
```diff
@@ -189,6 +139,69 @@ <h5>Related Resources</h5>
...
   </div>
 </div>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="WebAPI">UGRC API</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <p>
+      {% include fa_icon.html api=true class="svg-inline--fa fa-w-20 fa-4x
+      fa-pull-left" %} The UGRC API is an http enabeled service for accessing (via the internet) the geospatial data that we store in the State Geographic Information Datasource (SGID). The endpoints provided by the API are a great way to add geospatial functionality to your web pages and applications or desktop workflows. You will be able to geocode addresses and search all the data in the SGID. To get started,
```
Comment (jacobdadams) [inline data/address-geocoders-locators/index.html:149]:
```suggestion
      fa-pull-left" %} The UGRC API is an HTTP-enabeled service for accessing our geospatial data stored in the State Geographic Information Datasource (SGID) over the internet. The endpoints provided by the API are a great way to add geospatial functionality to your web pages and applications or desktop workflows. You will be able to geocode addresses and search all the data in the SGID. To get started,
```

File: data/address/index.html (hunk 1 above)
Comment (jacobdadams) [inline data/address/index.html]:
```suggestion
    <td>The recommended (and easy!) way to geocode a table of Utah addresses without any other software.</td>
```

**Review by jacobdadams — APPROVED** [review]
I like the organization and structuring- it puts all the different tools into perspective.

I've added a few suggestions that I think make it flow better, but you can take them or leave them.

File: data/address/index.html (hunk 1 above)
Comment (gregbunce) [reply data/address/index.html]: yes, i was going with the idea that as scott said, "if you are already working in Pro, masquerade would be easier than downloading the API client"

File: _posts/2021-11-29-introducing-the-official-ugrc-api-client.md (hunk 3 above)
Comment (steveoh) [reply _posts/2021-11-29-introducing-the-official-ugrc-api-client.md:30]: good catch.

File: data/address-geocoders-locators/index.html [hunk 5]
```diff
@@ -52,89 +59,32 @@
...
-      <a href="https://developer.mapserv.utah.gov/StartupGuide"
-        >read the getting started guide</a
-      >. There are also examples of geocoding in
-      <a href="https://github.com/agrc/GeocodingSample"
-        >multiple programming languages</a
-      >.
+      <img src="{% link images/api-client/web_api_logo.png %}" width="92" class="inline-text-left" alt="UGRC API logo" loading="lazy"/>The UGRC API Client is a cross-platform desktop geocoding tool designed to carefully and successfully guide you through your geocoding tasks. This app is our most user friendly geocoding client. The app will geocode <code>*.csv</code> files. It will require no license or subscription to use. It will automatically download updates as soon as they are available to stay current and add new features.
```
[the remark this answers is not in the document]
Comment (steveoh) [reply data/address-geocoders-locators/index.html]: i took a few things from this. thanks!

File: data/address-geocoders-locators/index.html [hunk 6]
```diff
@@ -52,89 +59,32 @@
...
-      <a href="https://developer.mapserv.utah.gov/StartupGuide"
-        >read the getting started guide</a
-      >. There are also examples of geocoding in
-      <a href="https://github.com/agrc/GeocodingSample"
-        >multiple programming languages</a
-      >.
+      <img src="{% link images/api-client/web_api_logo.png %}" width="92" class="inline-text-left" alt="UGRC API logo" loading="lazy"/>The UGRC API Client is a cross-platform, stand-alone desktop geocoding tool designed to carefully guide you step-by-step through your geocoding tasks. This app is our most user friendly geocoding client for geocoding <code>*.csv</code> files. It does not require a license or subscription to use and automatically download updates as soon as they are available to stay current and add new features.
```
Comment (jacobdadams) [inline data/address-geocoders-locators/index.html:69]:
```suggestion
      <img src="{% link images/api-client/web_api_logo.png %}" width="92" class="inline-text-left" alt="UGRC API logo" loading="lazy"/>The UGRC API Client is a cross-platform, stand-alone desktop geocoding tool designed to carefully guide you step-by-step through your geocoding tasks. This app is our most user friendly geocoding client for geocoding <code>*.csv</code> files. It does not require a license or subscription to use and automatically downloads updates as soon as they are available to stay current and add new features.
```

I missed an `s` the first time around.

**Review by jacobdadams — DISMISSED** [review]
I missed an `s` on the first suggestion. Mea culpa.

**Pull request merged**